kura/org.eclipse.kura.core.comm/src/main/java/org/eclipse/kura/core/comm/CommConnectionImpl.java

```diff
@@ -54,42 +56,37 @@
         }
     }
 
-    private final String m_port;
-    private final int m_baudRate;
-    private final int m_dataBits;
-    private final int m_stopBits;
-    private final int m_parity;
-    private final int m_flowControl;
-    private final int m_timeout;
 
-    private final CommURI m_commUri;
-    private SerialPort m_serialPort;
-    private InputStream m_inputStream;
-    private OutputStream m_outputStream;
+    private final CommURI commUri;
+    private SerialPort serialPort;
+    private InputStream inputStream;
+    private OutputStream outputStream;
 
     public CommConnectionImpl(CommURI commUri, int mode, boolean timeouts)
             throws IOException, NoSuchPortException, PortInUseException {
-        this.m_commUri = commUri;
-        this.m_serialPort = null;
-        this.m_port = this.m_commUri.getPort();
-        this.m_baudRate = this.m_commUri.getBaudRate();
-        this.m_dataBits = this.m_commUri.getDataBits();
-        this.m_stopBits = this.m_commUri.getStopBits();
-        this.m_parity = this.m_commUri.getParity();
-        this.m_flowControl = this.m_commUri.getFlowControl();
-        this.m_timeout = this.m_commUri.getTimeout();
+        this.commUri = commUri;
```

Comment:
But I would wait for Java 7 and use some more common Objects.* checks instead. I really hope we can at least advance to Java 7 in this release.